### PR TITLE
SONARIAC-286 [TF] Issue highlighting on attributes should be consistent

### PR DESCRIPTION
**Pull request by nils-werner-sonarsource**

### CODE REVIEW

File: iac-extensions/terraform/src/test/resources/checks/DisabledLoggingCheck/test_docdb_cluster.tf [hunk 1]
```diff
@@ -1,11 +1,12 @@
 resource "aws_docdb_cluster" "sensitive_exports_value" {
   cluster_identifier = "DB Cluster With Logs"
   enabled_cloudwatch_logs_exports = ["profiler"] # Noncompliant {{Make sure that disabling logging is safe here.}}
-                                  # ^^^^^^^^^^^^
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 }
 
 # Noncompliant@+1 {{Omitting enabled_cloudwatch_logs_exports makes logs incomplete. Make sure it is safe here.}}
 resource "aws_docdb_cluster" "missing_exports_value" {
+  #      ^^^^^^^^^^^^^^^^^^^
```
Comment (pierre-loup-tristant-sonarsource) [inline iac-extensions/terraform/src/test/resources/checks/DisabledLoggingCheck/test_docdb_cluster.tf:9]: Minor: it doesn't look like this test case is in the scope of [SONARIAC-286](https://jira.sonarsource.com/browse/SONARIAC-286)

File: iac-extensions/terraform/src/test/resources/checks/ClearTextProtocolsCheck/test_ecs_task_definition.tf [hunk 2]
```diff
@@ -3,13 +3,15 @@ resource "aws_ecs_task_definition" "ecs_task" {
   volume {
     # Noncompliant@+1 {{Omitting transit_encryption enables clear-text traffic. Make sure it is safe here.}}
     efs_volume_configuration {
+   #^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (pierre-loup-tristant-sonarsource) [inline iac-extensions/terraform/src/test/resources/checks/ClearTextProtocolsCheck/test_ecs_task_definition.tf:6]: Minor: it doesn't look like this test case is in the scope of [SONARIAC-286](https://jira.sonarsource.com/browse/SONARIAC-286)

**Review by pierre-loup-tristant-sonarsource — APPROVED** [review]
LGTM.
I left two minor comments.

**Comment by andrea-guarino-sonarsource** [comment]
LGTM! A minor remark on the ticket: IMO the ticket is a bit too much technical in its title and its content. I would suggest either to keep it as an implementation ticket and change its content to be more "functional" or to change the ticket type to "task".

**Review by andrea-guarino-sonarsource — APPROVED** [review]

**Pull request merged**